_targets.R

```diff
@@ -11,22 +11,21 @@ if(FALSE){ # Repeated builds can it GitHub API limit, set to TRUE to check for p
   remotes::install_dev("cyclestreets")
   remotes::install_github("dabreegster/odjitter", subdir = "r")
   remotes::install_github("ropensci/stplanr")# Improved overline
-  remotes::install_cran("targets")
 }
 
 library(targets)
 library(magrittr) # Light load of %>%
 library(sf)
 library(future) # Needed for multi-core running
 library(future.callr)
-
+library(stplanr)
+library(dplyr)
 # Set target options:
 pkgs = packages = c(
-  "tibble","zonebuilder","dplyr","stplanr","lubridate",
-  "cyclestreets","stringr","sf","tidyr","data.table",
+  "tibble","zonebuilder","dplyr","lubridate",
```

Comment:
Important change removes CRAN version of stplanr. Heads-up @mem48 and @joeytalbot this fixes the build.